### PR TITLE
[NO-CI] Update search results loaded event

### PR DESCRIPTION
**Pull request by esezen**

### CODE REVIEW

File: src/modules/tracker.js [hunk 1]
```diff
@@ -657,21 +656,19 @@ class Tracker {
           sortBy = sort_by,
           selected_filters,
           selectedFilters = selected_filters,
-          url = helpers.getWindowLocation()?.href || 'N/A',
+          url,
           section,
           analyticsTags,
-          resultCount = numResults || result_count || items?.length || 0,
```
Comment (jjl014) [inline src/modules/tracker.js:663]: Should we keep this change as well?

**Review by jjl014 — CHANGES_REQUESTED** [review]
Lots of changes in this PR, but things are mostly looking good from what I can tell. I did leave some comments around a few failing tests and some things that we may want to keep around. Lmk what you think. Thanks!

It might also be a good idea to get another review of the changes just in case I missed anything. 👍

File: spec/src/modules/tracker.js [hunk 2]
```diff
@@ -2029,10 +2029,10 @@ describe(`ConstructorIO - Tracker${bundledDescriptionSuffix}`, () => {
       });
 
       // eslint-disable-next-line max-len
-      expect(tracker.trackSearchResultsLoaded(term, snakeCaseParameters)).to.equal(true);
+      expect(tracker.trackSearchResultsLoadedV2(term, snakeCaseParameters)).to.equal(true);
     });
 
-    it('Should respond with a valid response when term and required parameters are provided', (done) => {
+    it('V2 Should respond with a valid response when term and required parameters are provided', (done) => {
```
[the remark this answers is not in the document]
Comment (esezen) [reply spec/src/modules/tracker.js:2035]: Fixed

File: src/modules/tracker.js [hunk 3]
```diff
@@ -1428,6 +1500,7 @@ class Tracker {
       const {
         section = 'Products',
         result_count,
+        resultCount = result_count,
```
Comment (jjl014) [inline src/modules/tracker.js:1503]:
```suggestion
        resultCount = result_count ?? items?.length ?? 0,
```

**Review by jjl014 — APPROVED** [review]
LGTM!

Just left one minor comment/suggestion, but I think this is ready to go. Thank you for working on this!

**Pull request merged**